src/framework/components/anim/component.js

```diff
@@ -213,6 +214,14 @@ class AnimComponent extends Component {
         return null;
     }
 
+    get consumedTriggers() {
+        return this._consumedTriggers;
+    }
+
+    set consumedTriggers(value) {
+        this._consumedTriggers = value;
+    }
+

```

Comment:
it seems the setter and getter does not do anything extra in this case, so I'd suggest to just have this.consumerTriggers = {}; in constructor and remove these. They can be always added when there's a need for them - you'd just rename the member in constructor by prefixing it with '_' , and implement setter / getter.